catalog/spatial/csw/spatial-csw-source-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/source/AbstractCswSource.java

```diff
@@ -1644,6 +1685,232 @@ public void setMetacardTypes(List<MetacardType> types) {
         this.metacardTypes = types;
     }
 
+    @Override
+    public CreateResponse create(CreateRequest createRequest) throws IngestException {
```

Comment:
This shouldn't even be attempted if the service we are connecting to doesn't support it. That's the point of obtaining the Capabilities and dynamically determining what operations it supports. We should not assume that's it's safe to send these types of requests to a service that doesn't support it.